src/Simulation/Simulation.jl

```diff
@@ -0,0 +1,178 @@
+using JSON, DataFrames, LightGraphs, PathDistribution
```

Comment:
I mentioned this above but I think I'd wrap this as a module and include the other files in it. What do you think?

Snake case variables and functions (sorry, last one 😆)